templates/v2/reset_password/beneficiary/reset_password.html.twig

```diff
@@ -0,0 +1,5 @@
+{% extends 'v2/pro_management_layout.html.twig' %}
+
+{% block title %}{{ 'reset_a_password'|trans }}{% endblock %}
+{% block subTitle %}{{ 'reset_beneficiary_password'|trans({'%username%': beneficiary.fullName})|upper }}{% endblock %}
```

Comment:
Maybe we could name this template "base" or "layout", cause it has no purpose on its own ?